CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 0.6.0
+
+ * Implement `manageRoutes` to enable the cloud route controller
+   - This will make sure that `VNIC`s have `SkipSourceDestCheck`
```

Comment:
updating VNICs is going to cause weirdness/competing with provisioning tools like Terraform. IMO, it's better to recommend how nodes should be provisioned instead of trying to reconcile them to avoid conflicts like this.